examples/feeds/create_feed_item_set.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Creates a new feed item set for a specified feed.
+
+The feed must belong to either a Google Ads location extension or an affiliate
+extension. This is equivalent to a "location group" in the Google Ads UI. See
+https://support.google.com/google-ads/answer/9288588 for more detail.
+"""
+
+import argparse
+import sys
+import uuid
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, feed_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        feed_id: the ID for a Feed belonging to the given customer.
+    """
+    feed_item_set_service = client.get_service(
+        "FeedItemSetService", version="v6"
+    )
+    feed_item_set_operation = client.get_type(
+        "FeedItemSetOperation", version="v6"
+    )
+
+    feed_item_set = feed_item_set_operation.create
+    feed_resource_name = client.get_service(
+        "FeedService", version="v6"
+    ).feed_path(customer_id, feed_id)
+    feed_item_set.feed = feed_resource_name
+    feed_item_set.display_name = "Feed Item Set #%s" % uuid.uuid4()
+
+    """A feed item set can be created as a dynamic set by setting an optional
```

Comment:
It is confusing mixing multi-line comments followed by # comments followed by multiline commented out blocks of code. I suggest you use only # comments in this code block per https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments

Reply:
Agree, and same applies for the lines below as well.

Reply:
Ack. I was following the pattern set in the Golden but it makes sense that commenting practices wouldn't be consistent across our libraries. I'll make sure to consult the styleguide next time :)